src/main/java/org/testng/internal/TestResult.java

```diff
@@ -39,6 +39,11 @@ public TestResult() {
 

```

Comment:
Is it possible to remove this constructor?

Reply:
No, it can't be done because its being used in org.testng.internal.Invoker#invokeMethod

Reply:
ok